consensus/consensus.go

```diff
@@ -54,6 +54,10 @@ type ChainHeaderReader interface {
 	GetHighestVerifiedHeader() *types.Header
 }
 
+type VotePool interface {
+	FetchVoteByHash(blockHash common.Hash) []*types.VoteEnvelope
```

Comment:
`FetchVoteByHash` is misleading,   `FetchVoteByBlockHash` is better.

Reply:
Updated